mne/channels/tests/test_montage.py

```diff
@@ -187,16 +187,14 @@ def test_montage():
     Fp1 -95.0 -31.0 -3.0
     AF7 -81 -59 -3
     AF3 -87 -41 28
-    nasion -91 0 -42
-    lpa 0 -91 -42
-    rpa 0 91 -42
+    FidNz -91 0 -42
+    FidT9 0 -91 -42
+    FidT10 0 91 -42
```

Comment:
Looks like this is expected, too

http://wiki.besa.de/index.php?title=Electrodes_and_Surface_Locations